fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -785,7 +785,7 @@ public void transferToNonLeader(FrontendNodeType newType) {
         getConfigRefreshDaemon().registerListener(() -> {
             try {
                 if (Config.max_broker_load_job_concurrency != loadingLoadTaskScheduler.getCorePoolSize()) {
-                    loadingLoadTaskScheduler.setCorePoolSize(Config.max_broker_load_job_concurrency);
+                    loadingLoadTaskScheduler.setPoolSize(Config.max_broker_load_job_concurrency);
```

Comment:
maybe just update the MaximumPoolSize? corePoolSize is the long time resident thread number.